doc/sphinx/io.rst

```diff
@@ -150,13 +150,15 @@ re-registered when the same checkpoint id is used later.
 Following the example above, the next example loads the last checkpoint,
 restores the state of all checkpointed objects and registers a signal.
 
-.. code:: python
+.. code::
+
     import espressomd from espressomd import checkpointing import signal
 
-    checkpoint = checkpointing.Checkpointing(checkpoint\_id=“mycheckpoint”)
+    checkpoint = checkpointing.Checkpointing(checkpoint_id=“mycheckpoint”)
```

Comment:
There are smart quotes used here (`“”`) instead of straight ones (`""`), which makes this invalid Python code. Since you're touching the line anyway, please fix this.